docs/data-source-api.md

```diff
@@ -176,6 +176,11 @@ const rawData = await dataSource.query(
     'SELECT * FROM USERS WHERE name = @0 and age = @1',
     [ 'John', 24 ]
 )
+
+-   `sql` - Executes a raw SQL query using template literals.
+
+```typescript
+const rawData = await dataSource.sql`SELECT * FROM USERS WHERE name = ${'John'} and age = ${24}`
 ```
```

Comment:
We're missing the closing backticks on the code block above this. And let's link to the new doc page.

````suggestion
```

-   `sql` - Executes a raw SQL query using template literals.

```typescript
const rawData = await dataSource.sql`SELECT * FROM USERS WHERE name = ${'John'} and age = ${24}`
```

Learn more about using the [SQL Tag syntax](sql-tag.md).
````